### PR TITLE
Logging and env vars

### PR DESCRIPTION
**Pull request by fgouteroux**
Hello @ricoberger 

First, thank for your work.

I want to add some enhancement logging, support windows event log when running as a service, and support environment vars in script config. Could you have a look on this, please ?

### CODE REVIEW

File: pkg/exporter/scripts.go [hunk 1]
```diff
@@ -56,8 +57,14 @@ func runScript(timeout float64, enforced bool, args []string) (string, int, erro
 	}
 	cmd = exec.CommandContext(ctx, args[0], args[1:]...)
 
-	if timeout > 0 {
+	// Set environments variables
+	if len(env) > 0 {
+		cmd.Env = env
+	} else {
 		cmd.Env = os.Environ()
+	}
```
Comment (ricoberger) [inline pkg/exporter/scripts.go:65]: Maybe `os.Environ()` can always be used and then the additional environment variables can be appended, sth. like

```suggestion
	cmd.Env = os.Environ()
	if len(env) > 0 {
		cmd.Env = append(cmd.Env, env...)
	}
```

**Comment by ricoberger** [comment]
Hi @fgouteroux awesome work, thank you very much 🙂.

Two smaller things:
1. Maybe `os.Environ()` can always be used, see my other comment
2. Can you please add the `env` key to the yaml schema in the readme (line 90). Maybe a short note in the sentence after the schema in the readme makes also sense.

**Comment by fgouteroux** [comment]
@ricoberger Thanks, I applied your suggestions.

**Comment by ricoberger** [comment]
Hi @fgouteroux thanks again 🙂. I published a new version [v2.8.0](https://github.com/ricoberger/script_exporter/releases/tag/v2.8.0) with your changes.

**Pull request merged**